commands/build.go

```diff
@@ -91,9 +94,39 @@ func preRunBuild(cmd *cobra.Command, args []string) error {
 		buildArgMap = mapped
 	}
 
+	buildLabelMap, err = parseBuildLabel(buildLabels)
+
 	return err
 }
 
+func parseBuildLabel(args []string) (map[string]string, error) {
```

Comment:
Does this logic already exist for the deploy labels? If its close enough can it be pulled up and reused.

Reply:
👍 I suspect it would do.. I agee with refactoring or re-using existing code for this.

Reply:
What bit is different? can you send a quick link to it here?

Reply:
https://github.com/openfaas/faas-cli/blob/bff2abda15f021927d03339f746227a0651bd5de/commands/deploy.go#L422-L442

Reply:
From looking at this, I would say we could use parseMap from the commands package?

Reply:
I have updated the code to use `parseMap`

Reply:
Thanks